layers/+lang/dart/packages.el

```diff
@@ -0,0 +1,76 @@
+;;; packages.el --- dart layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2012-2018 Sylvain Benner & Contributors
+;;
+;; Author: Bruno Tavares <connect+spacemacs@bltavares.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+;;; Code:
+
+;; TODO: offer lsp integration
+(defconst dart-packages
+  '(dart-mode
+    flycheck))
```

Comment:
Flycheck is an optional dependency. So, I would suggest you to remove this line.

Reply:
I'm not sure how dependencies work, but looking at other layers that have flycheck integration, all of them declares flycheck on the packages list.

I was mostly following the same setup, as the documentation shows as well.

Reply:
Its fine then. I didn't know about that.